tests/scripts/run-xunit-perf.cmd

```diff
@@ -18,11 +18,14 @@ setlocal
   set USAGE_DISPLAYED=
   set SHOULD_UPLOAD_TO_BENCHVIEW=
   set BENCHVIEW_PATH=
+  set COLLECTION_FLAGS=stopwatch
```

Comment:
If we do that then the default when you call run-xunit-perf.cmd will not run stopwatch, and I thought that was something that we wanted.  @valenis